src/components/organisms/SearchBar.vue

```diff
@@ -46,67 +55,46 @@ export default defineComponent({
     <v-card-title>
       Sample Project
     </v-card-title>
-    <TabToolBar
+    <!-- Will be needed in second iteration -->
+    <!-- <TabToolBar
       v-model="activeTab"
       :items="tabData"
       color="blue-grey darken-2"
       flat
-    />
+    /> -->
     <v-form @submit.prevent="$emit('input', inputs)">
-      <v-row
+      <!-- Will be needed in second iteration -->
+      <!-- <LatLongForm
         v-if="activeTab === 0"
-        no-gutters
-        justify="center"
-      >
-        <v-col
-          cols="5"
-          class="mr-3 mt-3"
-        >
-          <v-text-field
-            v-model.number="inputs.longitude"
-            label="Longitude(deg)"
-            outlined
-          />
-        </v-col>
-        <v-col
-          cols="5"
-          class="ml-3 mt-3"
-        >
-          <v-text-field
-            v-model.number="inputs.latitude"
-            label="Latitude(deg)"
-            outlined
-          />
-        </v-col>
-        <v-col
-          cols="9"
-          class="pl-1 pr-4"
-        >
-          <v-text-field
-            v-model.number="inputs.height"
-            label="Height (m)"
-            outlined
-          />
-        </v-col>
-        <v-btn
-          type="submit"
-          color="teal accent-4"
-          min-height="56px"
-          max-width="56px"
-          min-width="unset"
-        >
-          <v-icon
-            large
-          >
-            mdi-crosshairs-gps
-          </v-icon>
-        </v-btn>
-      </v-row>
+      />
+      <BoundingBoxForm
+        v-if="activeTab === 1"
+      /> -->
+      <ToolBar
+        text="Geo Json"
+        color="blue-grey darken-2"
+        flat
+      />
+      <GeoJsonForm />
     </v-form>
     <ToolBar
-      :text="text"
+      text="Other Params"
       color="blue-grey darken-2"
       flat
     />
+    <v-row
```

Comment:
Will we eventually other "Other Params" besides Predicate? If so, I think it would be nice to refactor this into it's own component to be consistent with the other forms; this would also raise the same question about the `ToolBar` component that I mentioned above for `GeoJsonForm`.

Reply:
Good point, I agree that this isn't really a toolbar. In this case I am sort of leaning on the mock-ups. I am not sure if this is best for consistency or maybe vue hierarchy. I am fine with changing it just not sure which is best here.
There will be a 'Search' (submit) button in this section but perhaps 'Other Params' is a misnomer then. @banesullivan  Will there eventually be other params here?